src/houses/KM_Houses.pas

```diff
@@ -465,6 +465,8 @@ procedure TKMHouse.DemolishHouse(aFrom: TKMHandIndex; IsSilent: Boolean = False)
 
 
 //Used by MapEditor
+//Return True, if house was set to new position successfully
+//aConsiderEntranceOffset - do consider house entrance offset while checking CanAddHousePlan and while set house to new position
```

Comment:
Old comment about aConsiderEntranceOffset. Also it says it returns true but it's a procedure?